src/app/src/main/kotlin/org/vaccineimpact/api/app/app_start/Router.kt

```diff
@@ -77,13 +79,33 @@ class Router(val config: RouteConfig,
         val controllerName = endpoint.controllerName
         val actionName = endpoint.actionName
 
-        val controllerType = Class.forName("org.vaccineimpact.api.app.controllers.${controllerName}Controller")
-
-        val controller = controllerType.getConstructor(ActionContext::class.java, Repositories::class.java)
-                .newInstance(context, repositories) as Controller
```

Comment:
These two lines have been pulled out into `instantiateController`. This move may obscure the fact that there is one small change in functionality: It now looks for a three argument constructor. The new argument in `WebTokenHelper`.